src/components/Article/Article.css

```
@@ -1 +1,32 @@
 /* Put article styles here */
+
+.article {
```

Remove redundant code. Check it everywhere, please 
```suggestion
.article {
```